tests/regression/46-apron2/02-localization-hh.c

```diff
@@ -0,0 +1 @@
+../34-localwn_restart/04-hh.c
```

Comment:
Yup, this is actually a symlink to that file, where it's also mentioned so that it's not changed in one place without realizing it changes something else as well:
```c
// This is part of 34-localization, but also symlinked to 36-apron.
```

It's because it has relational invariants, so it needs to be in an apron group, but it's also closely related to the localization stuff, so the actual file is still there.